api/nodes/serializers.py

```diff
@@ -1533,12 +1525,14 @@ def enforce_metadata_or_registration_responses(self, metadata=None, registration
                 ' `registration_responses` as `registration_metadata` will be deprecated in the future.',
             )
 
+    def get_node(self, validated_data=None):
+        return self.context['view'].get_node()
+
     def create(self, validated_data):
         initiator = get_user_auth(self.context['request']).user
-        node = self.context['view'].get_node()
+        node = self.get_node(validated_data)
         # Old workflow - deeply nested
         metadata = validated_data.pop('registration_metadata', None)
-        # New workflow - less nesting
```

Comment:
Add back - removed in merge with schemablocks

Reply:
🆗